app/src/org/commcare/views/media/MediaLayout.java

```diff
@@ -185,41 +185,47 @@ private void addAudioVideoButtonsToView(RelativeLayout questionTextPane) {
         // Add the audioButton and videoButton (if applicable) and view
         // (containing text) to the relative layout.
         if (audioButton != null) {
-            audioParams.addRule(RelativeLayout.ALIGN_PARENT_RIGHT);
-            textParams.addRule(RelativeLayout.LEFT_OF, audioButton.getId());
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
                 audioParams.addRule(RelativeLayout.ALIGN_PARENT_END);
                 textParams.addRule(RelativeLayout.START_OF, audioButton.getId());
+            } else {
```

Comment:
All of these "else" clause moves are no-ops that are here to be more explicit about the layout.